data/apps.ts

```diff
@@ -233,7 +234,13 @@ export const apps: appsList = {
       icon: feather,
       url: "https://apps.apple.com/app/feather-for-mastodon/id6446263061",
       paid: false,
-    }
+    },
+    {
+      released_on: "Aug 10, 2023",
+      name: "SoraSNS",
+      icon: sora,
+      url: "https://apps.apple.com/app/sora-for-mastodon-bluesky/id6450969760?platform=iphone",
+    },
```

Comment:
Please add a value for `paid` here (`true` or `false`), thank you.

Reply:
Thank you very much for your comment! 👍 I have added the required value.